spec/memoize_query_spec.rb

```diff
@@ -233,11 +234,21 @@ def expect_to_eq_with_or_without_redis
       end
 
       it 'recalculates after import' do
-        if Site.respond_to?(:import)
-          records = 5.times.map { Site.new(a: 2) }
-          Site.import(records)
-          expect(Site.a_count(2)).to eq(7)
+        records = 5.times.map { Site.new(a: 2) }
```

Comment:
This tests that the invalidation still works, but doesn't necessarily test the optimization that we're not invalidating the entire model anymore right? Is there any way we can do that?